infra/blueprint-test/test/policy-library/lib/constraints.rego

```diff
@@ -0,0 +1,36 @@
+#
```

Comment:
Can we move this policy-lib dir under the example dir?

Reply:
fixed